docs/developer/web-ui.rst

```diff
@@ -251,3 +251,61 @@ displayed on our UI.
 
 You can use any **font-awesome icon**. Kytos ui interface will read that icon
 and create a new button into the **k-menu-bar**.
+
+
+Toolbar Component Formats
+=========================
+
+A Toolbar component has content defined by the NApps in case it needs some
+user input and can exist in two format versions: compressed and expanded. You
+can learn more about this at the :doc:`./web-ui` page.
+
+New Toolbar Components that can be displayed in compressed and expanded
+formats must declare style attributes for both versions in the style tag.
+For the contents of the toolbar, Kytos Input Components already handle both
+formats. For new Input Components, the developer must declare the styles for
+the formats.
+
+For the compressed view the developer must declare the styles in the class
+selector called ``compacted``.
+
+Below is an example of usage in the
+`Input <https://github.com/kytos/ui/blob/1.3.1/src/components/kytos/inputs/Input.vue#L104>`_ component:
```

Comment:
It can be cleaner if you [separate the link and target definition](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks).